doc/Manuals/ig-ss_x-road_v6_security_server_installation_guide_for_rhel7.md

```diff
@@ -0,0 +1,297 @@
+
+
+## 3 Security Server Initial Configuration
+
+During the security server initial configuration, the server’s X-Road membership information and the software token’s PIN are set.
+
+
+### 3.1 Prerequisites
+
+Configuring the security server assumes that the security server owner is a member of the X-Road.
+
+
+### 3.2 Reference Data
+
+ATTENTION: Reference items 2.1 - 2.3 in the reference data are provided to the security server owner by the X-Road central’s administrator.
+
+The security server code and the software token’s PIN will be determined during the installation at the latest, by the person performing the installation.
+
+ Ref  |                                                   | Explanation
+ ---- | ------------------------------------------------- | --------------------------------------------------
+ 2.1  | &lt;global configuration anchor file&gt; or &lt;URL&gt; | Global configuration anchor file
+ 2.2  | E.g.<br>GOV - government<br> COM - commercial     | Member class of the security server's owner
+ 2.3  | &lt;security server owner register code&gt;       | Member code of the security server's owner
+ 2.4  | &lt;choose security server identificator name&gt; | Security server's code
+ 2.5  | &lt;choose PIN for software token&gt;             | Software token’s PIN
+
+
+### 3.3 Configuration
+
+To perform the initial configuration, open the address
+
+    https://SECURITYSERVER:4000/
+
+in a Web browser (**reference data: 1.8; 1.6**). To log in, use the account name chosen during the installation (**reference data: 1.3).**
+
+Upon first log-in, the system asks for the following information.
+
+-   The global configuration anchor file (**reference data: 2.1**).
+
+    **Please verify anchor hash value with the published value.**
+
+If the configuration is successfully downloaded, the system asks for the following information.
+
+-   The security server owner’s member class (**reference data: 2.2**).
+
+-   The security server owner’s member code (**reference data: 2.3**).
+    If the member class and member code are correctly entered, the system displays the security server owner’s name as registered in the X-Road center.
+
+-   Security server code (**reference data: 2.4**), which is chosen by the security server administrator and which has to be unique across all the security servers belonging to the same X-Road member.
+
+-   Software token’s PIN (**reference data: 2.5**). The PIN will be used to protect the keys stored in the software token. The PIN must be stored in a secure place, because it will be no longer possible to use or recover the private keys in the token once the PIN has been lost.
+
+
+## 4 Installation Error handling
+
+### 4.1 UI Does Not Respond or Returns an Error Message
+
```

Comment:
There can be also other reasons for the admin UI not responding (e.g. xroad-jetty is not running).
Disabling IPv6 can help if nginx resolves localhost to [::1] because xroad-jetty listens on the IPv4 localhost. However, changing "proxy pass http://localhost:8083;" to "proxy pass http://127.0.0.1:8083;" in /etc/xroad/nginx/default-xroad.conf and restarting nginx should be enough in that case (probably should make that default).

Reply:
Section 4.1 updated based on comments.